README.md

```diff
@@ -46,9 +46,10 @@ async def main():
     
     #replace "thing" with the actual name of the generic component in the machine config 
     led = Generic.from_robot(robot, "thing")
+    #decimal format green, 65280
     for color in [65280, 0]:
         #replace with the actual number of lights on the ring or strip
-        for pixel in range(7):
+        for pixel in range(24):
```

Comment:
Optional: do you think we can just pass this down from the attributes rather than hard-coding?  If you know how to do that quickly it might be worth doing.  otherwise this is fine and I can poke around at attributes again later.

Reply:
I don't know on top of mind, but agree that would be useful. If you take a look let me know how it goes

Reply:
👍